### ISSUE TITLE
defaultColumns / how to set width for id-column?

### ISSUE THREAD

**Issue by crazyx13th** [keystonejs/keystone#4588]
Hi There,

I'm trying to set the width for the "id", but nothing happend.
I used **id, _id, kex, __name__,** ... no luck

![grafik](https://user-images.githubusercontent.com/8912561/36896658-ce408956-1e13-11e8-909d-190ecbda09f4.png)

![grafik](https://user-images.githubusercontent.com/8912561/36896625-ad9673aa-1e13-11e8-9226-a25dd6cbfb81.png)

any idea?
thx!

Greetings
crazyx13th

**Comment by tmvanetten** [comment]
@crazyx13th the only way I know to set the column width is by percentage.

example `Statement.defaultColumns = 'id|10%, group, statement';`

**Comment by crazyx13th** [comment]
sure that it should work? What I wrote before, i tried: `Statement.defaultColumns` with `id, _id, kex, name`. Nothing worked. thx

**Comment by crazyx13th** [comment]
![grafik](https://user-images.githubusercontent.com/8912561/36973527-a90a0646-2073-11e8-98d6-cfb349c6f972.png)
![grafik](https://user-images.githubusercontent.com/8912561/36973523-a2bf2988-2073-11e8-91b0-b89b3ed56b01.png)

![grafik](https://user-images.githubusercontent.com/8912561/36973496-8b4dc412-2073-11e8-840d-1e38fa1d207a.png)

**Comment by rasmniel** [comment]
I can confirm that changing the width of the ID column ie `defaultColumns: 'id|10%'` does nothing.
the column does not seem to identify as `id`, `_id`, `ID`, etc.
It works as expected on other columns.

Currently, I would suggest a simple CSS hack injected using the Keystone property: `adminui custom styles`.

**Comment by autoboxer** [comment]
I found a simpler solution which works @crazyx13th.  Simply set the width values of the other columns, leaving the percentage space you want for the `id` column.  In my case I have three fields appear: `id`, `date`, `note`, for an `Internal Note` model I've created.  I set the default columns as follows (note that I'm leaving 10% of the total width for the `id` field):

```
InternalNotes.defaultColumns = 'date|20%, note|70%';
```

Which produces the following result:

![image](https://user-images.githubusercontent.com/2757601/56051365-56e7a580-5d1c-11e9-9578-44d915820942.png)

With a workaround present and with minimal changes being added to the Keystone 4 repo, I'm closing this out.
